utils/python/CIME/utils.py

```diff
@@ -149,8 +149,7 @@ def run_cmd(cmd, ok_to_fail=False, input_str=None, from_dir=None, verbose=None,
     if (arg_stderr is _hack):
         arg_stderr = subprocess.PIPE
 
```

Comment:
I plan to rework the logging facility a bit and will address this at that point.